altair/examples/line_chart_with_generator.py

```diff
@@ -0,0 +1,22 @@
+"""
+Line Chart with Sequence Generator
+----------------------------------
+This examples shows how to create multiple lines using the sequence generator.
+"""
+# category: line charts
+
+import altair as alt
+
+source = alt.sequence(start=0, stop=12.7, step=0.1, as_='x')
+
+alt.Chart(source).mark_line().transform_calculate(
+    sin='sin(datum.x)'
+).transform_calculate(
```

Comment:
You can put both ``sin`` and ``cos`` as arguments to a sinlge ``transform_calculate`` call.

Reply:
oh I didn't know that!